extensions/amp-analytics/0.1/variables.js

```diff
@@ -159,6 +159,11 @@ export class VariableService {
     /** @const @private {!./linker-reader.LinkerReader} */
     this.linkerReader_ = linkerReaderServiceFor(this.ampdoc_.win);
 
+    /** @const @private {!../../../src/service/url-replacements-impl.UrlReplacements} */
+    this.urlReplacementService_ = Services.urlReplacementsForDoc(
```

Comment:
We can't do this. Instead we'll have to call `Services.urlReplacementsForDoc(element)` for every element we want to expand. Because FIE doesn't have its own ampdoc, but it has its own urlReplacementService.

Reply:
Moved to get urlReplacementService from the element inside the `expandTemplate` call.